app/services/post_check_availability_task_service.rb

```diff
@@ -0,0 +1,60 @@
+class PostCheckAvailabilityTaskService < TaskService
+  STATUS_AVAILABLE, STATUS_UNAVAILABLE = %w[available unavailable].freeze
+  EVENT_AVAILABILITY_STATUS            = "availability_status".freeze
+  SERVICE_NAME                         = "platform.sources.status".freeze
+
+  def initialize(options)
+    super
+    @source = Source.find(options[:source_id]) || raise("source_id must be specified in options")
+    @task   = CheckAvailabilityTask.find(options[:task_id]) || raise("task_id must be specified")
+  end
+
+  def process
+    # TODO: need to add two columns to save available status
```

Comment:
@hsong-rh In the source object in the Sources API we have these extra fields
```
Source(id: integer, name: string, uid: string, created_at: datetime, updated_at: datetime, tenant_id: integer, source_type_id: integer, version: string, availability_status: string, imported: string, source_ref: string, last_checked_at: datetime, last_available_at: datetime)
```
We need these extra vars 
availability_status, last_checked_at, last_available_at